core/services/relay/evm/mercury/config_digest.go

```diff
@@ -61,10 +61,10 @@ func configDigest(
 		// assertion
 		panic("copy too little data")
 	}
-	if types.ConfigDigestPrefixEVM != 1 {
-		// assertion
-		panic("wrong ConfigDigestPrefix")
-	}
+	// if types.ConfigDigestPrefixMercury != 6 {
+	// 	// assertion
+	// 	panic("wrong ConfigDigestPrefix")
+	// }
```

Comment:
Can this be removed? If not, maybe a comment?

Reply:
Yeah I can add a comment here

Reply:
Done!